### PR TITLE
Multi player power ups

### PR DESCRIPTION
**Pull request by bryanvanwijk**
I created the class to controll the powerups in a multiPlayer Game.

### CODE REVIEW

**Comment by egedebruin** [comment]
Good work

File: src/test/java/spaceinvaders/group_22/PlayerTest.java [hunk 1]
```diff
@@ -19,7 +19,8 @@
 	@SuppressWarnings("checkstyle:magicnumber")   
 	public final void testGetSetSpaceShip() {
 		SpaceShip ship = new SpaceShip(10.0, 10.0, "testimage.png");
-		Player player = new Player(new Game(200 , 200));
+		Game game = new Game(200, 200);
+		Player player = new Player(game, game.getCanvasWidth() / 2);
```
Comment (Kingdorian) [inline src/test/java/spaceinvaders/group_22/PlayerTest.java]: A better way to do this is change getCanvasWitdth() to 100. Or create an int canvasWidth and also use it to create the game object.

**Comment by doriandekoning** [comment]
Seems fine!

**Comment by jochem725** [comment]
Looks great!

**Pull request merged**